### PR TITLE
[CIVisibility] Test visibility - Supported features table

### PR DESCRIPTION
**Pull request by tonyredondo**
### What does this PR do? What is the motivation?

This PR adds a new `Supported features` table to the `Test Visibility` section.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/tests/_index.md [hunk 1]
```diff
@@ -33,6 +33,24 @@ cascade:
...
+|-----------------------------------------|------|------|------------|--------|------|-------|-----------|
+| Accurate time/durations results         | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
+| Structured stack traces                 | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
+| Distributed traces on Integration tests | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
+| Agent based reporting                   | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
+| Agentless reporting                     | ✓    | ✓    | ✓          | ✓      | ✗    | ✓     | ✓         |
+| Test suite level visibility             | ✓    | ✓    | ✓          | ✓      | ✗    | ✓     | ✓         |
+| Manual API                              | ✓    | ✓    | ✓          | ✗      | ✗    | ✓     | ✗         |
+| Codeowner by test                       | ✓    | ✓    | ✓          |        |      | ✓     | ✗         |
+| Source code start/end reporting         | ✓    | ✓    | ✓          |        |      | ✓     | ✗         |
```
Comment (juan-fernandez) [inline content/en/continuous_integration/tests/_index.md:47]: I don't know if this suggestion breaks your formatting but I'd say it's important to note 
```suggestion
| Source code start/end reporting         | ✓    | ✓    | ✓ (only start)          |        |      | ✓     | ✗         |
```

**Review by juan-fernandez — APPROVED** [review]

File: content/en/continuous_integration/tests/_index.md [hunk 2]
```diff
@@ -33,6 +33,24 @@ cascade:
...
+| CI and Git info                         | ✓    | ✓    | ✓          | ✓      |      | ✓     | ✓         |
+| Git metadata upload                     | ✓    | ✓    | ✓          | ✓      |      | ✓     | ✓         |
+| Intelligent Test Runner                 | ✓    | ✓    | ✓          | ✓      |      | ✓     | ✗         |
+| Code coverage support                   | ✓    | ✓    | ✓          |        |      | ✓     | ✗         |
+| Benchmark tests support                 | ✓    | ✗    | ✗          | ✓      | ✗    | ✓     | ✗         |
+
```
Comment (anmarchenko) [inline content/en/continuous_integration/tests/_index.md:53]: Here is my understanding of how it looks like for ruby:
```suggestion

### Supported features
|                                         | .NET | Java | Javascript | Python | Ruby | Swift | JUnit Xml |
|-----------------------------------------|------|------|------------|--------|------|-------|-----------|
| Accurate time/durations results         | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
| Structured stack traces                 | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
| Distributed traces on Integration tests | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
| Agent based reporting                   | ✓    | ✓    | ✓          | ✓      | ✓    | ✓     | ✗         |
| Agentless reporting                     | ✓    | ✓    | ✓          | ✓      | ✗    | ✓     | ✓         |
| Test suite level visibility             | ✓    | ✓    | ✓          | ✓      | ✗    | ✓     | ✓         |
| Manual API                              | ✓    | ✓    | ✓          | ✗      | ✗    | ✓     | ✗         |
| Codeowner by test                       | ✓    | ✓    | ✓          |        | ✗    | ✓     | ✗         |
| Source code start/end reporting         | ✓    | ✓    | ✓          |        | ✗    | ✓     | ✗         |
| CI and Git info                         | ✓    | ✓    | ✓          | ✓      | ✓ 
```

File: content/en/continuous_integration/tests/_index.md (hunk 1 above)
Comment (tonyredondo) [reply content/en/continuous_integration/tests/_index.md]: done in https://github.com/DataDog/documentation/pull/19793/commits/2cc0d88ca951ceee6c73960eb3fce06a6f7d00bf

File: content/en/continuous_integration/tests/_index.md (hunk 2 above)
Comment (tonyredondo) [reply content/en/continuous_integration/tests/_index.md]: great thanks, done in https://github.com/DataDog/documentation/pull/19793/commits/2cc0d88ca951ceee6c73960eb3fce06a6f7d00bf

File: content/en/continuous_integration/tests/_index.md [hunk 3]
```diff
@@ -33,6 +33,25 @@ cascade:
...
+| Test suite level visibility             |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
+| Manual API                              |   ✓  |   ✓  |        ✓       |    ✗   |   ✗  |   ✓   |     ✗     |
+| Codeowner by test                       |   ✓  |   ✓  |        ✓       |        |   ✗  |   ✓   |     ✗     |
+| Source code start/end reporting         |   ✓  |   ✓  | ✓ (only start) |        |   ✗  |   ✓   |     ✗     |
+| CI and Git info                         |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✓     |
+| Git metadata upload                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
+| Intelligent Test Runner                 |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✗     |
+| Code coverage support                   |   ✓  |   ✓  |        ✓       |        |   ✗  |   ✓   |     ✗     |
+| Benchmark tests support                 |   ✓  |   ✗  |        ✗       |    ✓   |   ✗  |   ✓   |     ✗     |
+| Parameterized tests                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✗     |
```
Comment (romainkomorndatadog) [inline content/en/continuous_integration/tests/_index.md:53]:
```suggestion
| Accurate time/durations results         |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
| Structured stack traces                 |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
| Distributed traces on Integration tests |   ✓  |   ✓  |        ✓       |       ✗     |   ✓  |   ✓   |     ✗     |
| Agent based reporting                   |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✗     |
| Agentless reporting                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
| Test suite level visibility             |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
| Manual API                              |   ✓  |   ✓  |        ✓       |    ✗   |   ✗  |   ✓   |     ✗     |
| Codeowner by test                       |   ✓  |   ✓  |        ✓       |   ✗  |   ✗  |   ✓   |     ✗     |
| Source code start/end reporting         |   ✓  |   ✓  | ✓ (only start) |   ✗  |   ✗  |   ✓   |     ✗     |
| CI and Git info                         |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |     ✓     |
| Git metadata upload                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✓     |
| Intelligent Test Runner                 |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✗     |
| Code coverage support                   |   ✓  |   ✓  |        ✓       |    ✗  |   ✗  |   ✓   |     ✗     |
| Benchmark tests support                 |   ✓  |   ✗  |        ✗       |    ✓   |   ✗  |   ✓   |     ✗     |
| Parameterized tests                     |   ✓  |   ✓  |        ✓       |    ✓   |   ✗  |   ✓   |     ✗     |
```

Comment (tonyredondo) [reply content/en/continuous_integration/tests/_index.md]: done in [8c48292](https://github.com/DataDog/documentation/pull/19793/commits/8c4829261e3bd92e0ac78fee7bf69ed046e5b49b)

**Review by romainkomorndatadog — APPROVED** [review]

File: content/en/continuous_integration/tests/_index.md [hunk 4]
```diff
@@ -33,6 +33,25 @@ cascade:
 
 [Test Visibility][1] provides a test-first view into your CI health by displaying important metrics and results from your tests. It can help you investigate performance problems and test failures that concern you the most because you work on the related code, not because you maintain the pipelines they are run in.
 
+### Supported features
+|                                         | .NET | Java |   Javascript   | Python | Ruby | Swift |    JUnit Xml   |
+|-----------------------------------------|:----:|:----:|:--------------:|:------:|:----:|:-----:|:--------------:|
+| Accurate time/durations results         |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |        ✗       |
+| Structured stack traces                 |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |        ✗       |
+| Distributed traces on Integration tests |   ✓  |   ✓  |        ✓       |    ✗   |   ✓  |   ✓   |        ✗       |
+| Agent based reporting                   |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |        ✗       |
```
Comment (maycmlee) [inline content/en/continuous_integration/tests/_index.md:42]:
```suggestion
| Agent-based reporting                   |   ✓  |   ✓  |        ✓       |    ✓   |   ✓  |   ✓   |        ✗       |
```

**Review by maycmlee — COMMENTED** [review]
Just a couple of small suggestions!

**Review by maycmlee — APPROVED** [review]

**Pull request merged**